engine/src/main/scala/cromwell/engine/workflow/lifecycle/WorkflowExecutionActor.scala

```diff
@@ -95,6 +99,8 @@ object WorkflowExecutionActor {
     override val exceptionContext = s"WorkflowExecutionActor"
   }
 
+  val MaxRetries = ConfigFactory.load().getInt("system.maxRetries")
```

Comment:
It'd be good to either make sure this value exists at start-up time, or make this optional.

It'd be pretty awkward to have to wait until a workflow is submitted to find out the config file is invalid